tests/delete_regress/tests.py

```diff
@@ -366,3 +366,22 @@ def test_foreign_key_delete_nullifies_correct_columns(self):
         self.assertEqual(researcher1.secondary_contact, contact2)
         self.assertEqual(researcher2.primary_contact, contact2)
         self.assertIsNone(researcher2.secondary_contact)
+
+
+class DeleteDistinct(TestCase):
+    """Ticket #32433"""
```

Comment:
Since no queries are perform you can subclass `SimpleTestCase`. You also can drop the ticket reference as we've moved to only include them for cryptic errors that require a lot of context since nowadays most of these a tests are one `git blame` away from a link to their ticket.

```suggestion
class DeleteDistinct(SimpleTestCase):
```

Reply:
Sure thing, will do.